### ISSUE TITLE
creates unusable, un-deleteable files 

### ISSUE THREAD

**Issue by omniscopic** [latenitefilms/TransferToolbox#3]
I was excited to make this work and admit I might be doing something wrong but tried to follow the workflow precisely as instructed. I am using Sonoma 14.1.1 on a Macbook Pro with FCP 10.7.1 and FCP 1.3 for iPad Pro with iOS 17.4. 

I drop the Library into the TransferToolbox app and get a message about using the "Default Event Date" format (which I do), it eventually give me an export success message but when I try to transfer the .fcpproj file over to the iPad it give me a warning that the file cannot be moved.

`The operation can’t be completed because one or more required items can’t be found.
(Error code -43)`

Further, when I try to delete the file I get the same (Error code-43). I've tried everything to do this including rebooting in safe mode and through Terminal. 

ANY help here would be Much appreciated as I'd love to see this app work.

Thanks!

**Comment by latenitefilms** [comment]
Very strange! You're the first person to report something like this.

What format (ie APFS, HFS, etc) was the drive containing the Final Cut Pro (for Mac) Library? Was the exported iPad project on the same drive?

Did you have the Final Cut Pro library closed when you ran Transfer Toolbox?

Is all the media in the library internal?

Have you tried running Disk Utility Repair on the drive (https://support.apple.com/en-us/102611)?

**Comment by omniscopic** [comment]
Thanks for the response. You're asking the right questions. All media was housed internally on the Mac. I ran another test with a simple timeline. This time, the only difference was that I QUIT FCP before dropping the library into the Transfer Toolbox. It seems to have worked. In Serge's video he says "Close FCP". in the previous case (where it didn't work, I can't remember if I did close it). This time I QUIT it just to be sure. 

I'll run a few more tests with more complex edits. Thanks!

The 43 Error might also have something to do with the fact that my files are automatically backed up to a cloud drive and perhaps that introduced a corruption? I'll trouble shoot the 43 Error separately but just wanted to report my results for the knowledgebase. 

Thanks again!
